Tools/Firmware/CMakeLists.txt

```diff
@@ -38,6 +41,10 @@ include_directories(
     ${ROOT_DIR}/Boardfiles/${FOLDER_NAME}/Middlewares/Third_Party/FreeRTOS/Source/CMSIS_RTOS_V2
     ${ROOT_DIR}/Boardfiles/${FOLDER_NAME}/Middlewares/Third_Party/FreeRTOS/Source/portable/GCC/ARM_${PORTABLE_NAME}
     ${ROOT_DIR}/Boardfiles/${FOLDER_NAME}/Middlewares/Third_Party/FreeRTOS/Source/include
+    
+    ## Models Includes ##
+    ${ROOT_DIR}/Models
+    ${ROOT_DIR}/Models/${MODEL_NAME}
```

Comment:
whats going on here

Reply:
Not sure. I copied and pasted the file contents from the current main branch.

Reply:
Am I good to resolve this conversation?

Reply:
This is the Model Config code already in main.